src/javascript/_common/tmb.js

```diff
@@ -0,0 +1,278 @@
+        return 'svg'; // default
+    };
+
+    /**
+     * Get active sessions from TMB
+     * @returns {Promise<Object|null>} Active session data or null if failed
+     */
+    const getActiveSessions = async () => {
+        try {
+            const data = await requestSessionActive();
+            return data;
+        } catch (error) {
+            return null;
+        }
+    };
+
+    /**
+     * Process and store TMB account data
+     * @param {Object} activeSessions - Active session data from TMB
+     */
+    const processActiveSessions = async (activeSessions) => {
+        if (!activeSessions?.active || !activeSessions?.tokens) {
+            // reset login store when session is inactive
+            await TMB.handleTMBLogout();
+
+            return;
+        }
+
+        // Transform account data to SmartTrader format
+        const accounts = transformTMBAccounts(activeSessions);
+        
+        // Store accounts in localStorage using ClientBase format
+        LocalStore.setObject('client.accounts', accounts);
+
+        // Set the first account as active if no active account is set
+        const accountIds = Object.keys(accounts);
+        if (accountIds.length > 0) {
+            const currentActiveId = LocalStore.get('active_loginid');
+            if (!currentActiveId || !accounts[currentActiveId]) {
+                const firstAccountId = accountIds[0];
+                LocalStore.set('active_loginid', firstAccountId);
+                SessionStore.set('active_loginid', firstAccountId);
+            }
+        }
+
+        // Set logged_state cookie for cross-app compatibility
+        const currentDomain = getCurrentDomain();
+        if (supportedDomains.includes(currentDomain)) {
+            Cookies.set('logged_state', 'true', {
+                domain : currentDomain,
+                expires: 30,
+                path   : '/',
+                secure : true,
+            });
+        }
+
+        // Initialize Client with the new account data
+        Client.init();
+    };
+
+    /**
+     * Handle TMB login flow
+     * @returns {Promise<boolean>} True if login was successful, false otherwise
+     */
+    const handleTMBLogin = async () => {
+        try {
+            const activeSessions = await getActiveSessions();
+            
+            if (activeSessions?.active) {
+                await processActiveSessions(activeSessions);
+                
+                // Handle URL redirection if needed
+                if (typeof window !== 'undefined' &&
+                    window.location.pathname === '/' &&
+                    window.location.search) {
+                    // Clear search params after successful login
+                    const url = new URL(window.location.href);
+                    url.search = '';
+                    window.history.replaceState({}, '', url.toString());
+                }
+                
+                return true;
+            }
```

Comment:
else {
   // reset login store when session is inactive
            await TMB.handleTMBLogout();

            return;
}